libnavigation-core/src/main/java/com/mapbox/navigation/core/directions/session/DirectionsSession.kt

```diff
@@ -48,24 +46,6 @@ internal interface DirectionsSession {
      */
     fun cancelRouteRequest(requestId: Long)
 
-    /**
-     * Refresh the traffic annotations for a given [DirectionsRoute]
-     *
-     * @param route DirectionsRoute the direction route to refresh
-     * @param legIndex Int the index of the current leg in the route
-     * @param callback Callback that gets notified with the results of the request
-     */
-    fun requestRouteRefresh(
-        route: NavigationRoute,
-        legIndex: Int,
-        callback: NavigationRouterRefreshCallback
-    ): Long
-
-    /**
-     * Cancels [requestRouteRefresh].
-     */
-    fun cancelRouteRefreshRequest(requestId: Long)
```

Comment:
TBH I don't see any value in accessing router via direction session. This is a small step on the way to provide router directly.
@LukasPaczos , @RingerJK , what do you think about it?